tordnscrypt/src/main/res/values/strings.xml

```diff
@@ -15,18 +15,18 @@
     <string name="btnITPDStart" translatable="false">Start I2P</string>
     <string name="btnITPDStop" translatable="false">Stop I2P</string>
     <string name="tvITPDStop">I2P Stopped</string>
-    <string name="tvITPDRunning">I2P Running</string>
-    <string name="tvITPDStarting">I2P Starting</string>
-    <string name="tvITPDStopping">I2P Stopping</string>
-    <string name="tvITPDInstalling">I2P Installing</string>
+    <string name="tvITPDRunning">I2P Running…</string>
+    <string name="tvITPDStarting">I2P Starting…</string>
+    <string name="tvITPDStopping">I2P Stopping…</string>
+    <string name="tvITPDInstalling">I2P Installing…</string>
     <string name="tvITPDInstalled">I2P Installed</string>
-    <string name="tvITPDNotInstalled">I2P Not Installed</string>
-    <string name="tvITPDDefaultLog" translatable="false">Purple I2P version</string>
+    <string name="tvITPDNotInstalled">I2P is Not Installed</string>
+    <string name="tvITPDDefaultLog" translatable="false">Purple I2P, version</string>
 
-    <string name="swRunOnBoot">Start on Boot</string>
+    <string name="swRunOnBoot">Start with Device</string>
```

Comment:
```suggestion
    <string name="swRunOnBoot">Start along with Device</string>
```
The idea behind the change(s) is to be more informative for novice users that don't know what "boot" is.
There are more such entries. What do you think?